src/enums/pokeball.ts

```diff
@@ -3,5 +3,7 @@ export enum PokeballType {
   GREAT_BALL,
   ULTRA_BALL,
   MASTER_BALL,
-  // LUXURY_BALL,
+  // Temporarily used to represent epic tier items in the shop, not available to the player
+  // For that, it needs to be placed after the master ball
```

Comment:
NIT: Should the comment display on `inspect`? 
Then it should use 
```ts
/** */
``` 
Or the multi-line version of it